app/js/newTest.js

```diff
@@ -1,10 +1,26 @@
-document.querySelector('.new-test-button').addEventListener('click', function (x) {
-        let inputLength = document.querySelector('#addTestName').value.length
-        if (inputLength < 1 || inputLength > 255) {
-            alert('Computer says no.')
-        } else {
-            alert('Computer says yes.')
-        }
+var submit = document.querySelector('.new-test-button')
+var url = 'http://localhost:8080/test'
 
-        x.preventDefault()
-})
\ No newline at end of file
+submit.addEventListener('click', function (e) {
+    let inputLength = document.querySelector('#addTestName').value.length
+    if (inputLength < 1 || inputLength > 255) {
+        alert('Test name must be less than 250 characters')
+    } 
+    e.preventDefault()
+})
+
+async function postData(url, submit) {
+    submit.addEventListener('click', async function (e) {
+        e.preventDefault()
+        var test_id = document.querySelector('#addTestName').value
+        test_id = { "name": test_id }
```

Comment:
replace test_id with testObject throughout (or something similar)

Reply:
Not a test object, this is a part of the object that will make the whole test. 

The test object will be a full JSON file.